source/common/upstream/outlier_detection_impl.h

```diff
@@ -205,6 +213,15 @@ class DetectorHostMonitorImpl : public DetectorHostMonitor {
   void putResultWithLocalExternalSplit(Result result, absl::optional<uint64_t> code);
   std::function<void(DetectorHostMonitorImpl*, Result, absl::optional<uint64_t> code)>
       put_result_func_;
+
+  // Set of extension monitors.
+  ExtMonitorsSet monitors_set_;
+  // The following fields are reported when extension monitor is "tripped" and
+  // reports that a host where the extension monitor was attached should be
+  // ejected.
+  std::string failed_monitor_name_;
+  uint32_t failed_monitor_enforce_;
```

Comment:
where the `failed_monitor_enforce_` is used?

Reply:
It is used in a callback, but I am going to try to rework it.

Reply:
Those fields have been removed and `name` and `enforce` are passed by value in the event posted to the main thread.